keps/sig-network/563-dual-stack/README.md

```diff
@@ -1392,20 +1388,18 @@ This capability will move to stable when the following criteria have been met.
   N/A
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+  We did manual testing of a cluster turning it off and on to explore
+  disabled-with-data behavior.
+  [TODO] Describe outcome of testing
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
 fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  Enabling this without configuring the CLI options will not change any default
+  behavior.
```

Comment:
Removing template text and adding text from feedback - please check for accuracy.

Reply:
Feedback on https://github.com/kubernetes/enhancements/pull/2325/files#r568825561 was: "I wouldn't say "No" but instead say "Enabling this without configuring the CLI options will not change any default behavior." The rest of the statement is good."

Reply:
Yes, accurate.